### ISSUE TITLE
Will it work on 478.0.0.0.0?

### ISSUE THREAD

**Issue by Askhento** [IsmaelHG/UndervoltPatchMacBook#6]
<img width="361" alt="image" src="https://github.com/IsmaelHG/UndervoltPatchMacBook/assets/34323808/2e5c0d59-e588-4ed9-b42e-f9888c49d346">

**Comment by teqtic** [comment]
Yes, it should. I just completed the patch on my Big Sur 478.0.0.0 firmware.
